docs/reference/analysis/tokenfilters/edgengram-tokenfilter.asciidoc

```diff
@@ -173,6 +173,10 @@ See <<analysis-edgengram-tokenfilter-max-gram-limits>>.
 (Optional, integer)
 Minimum character length of a gram. Defaults to `1`.
 
+`preserve_original`::
+(Optional, boolean)
+If set to true then it would also emit the original token. Defaults to `false`.
```

Comment:
nit: wording might be better sth like "Emits original token then set to `true`. Defaults to `false`.

Reply:
changed to `Emits original token when set to `true`. Defaults to `false`.` --> notice changed to `when` from `then` in the suggested edit.